packages/components/Molecules/Form/fields/fieldRplchecklist.vue

```diff
@@ -2,7 +2,11 @@
   <!-- This component is deprecated, use ripple-select instead. -->
```

Comment:
As discussed the component was only deprecated for use in dropdowns, you can remove this comment now

Reply:
Thanks @dylankelly, the comment is removed now as well. 👍